--- wxWidgets/src/gtk/bitmap.cpp	2005-09-25 20:59:03.000000000 +0100
+++ wxWidgets-bmpfix/src/gtk/bitmap.cpp	2005-09-30 11:33:30.000000000 +0100
@@ -1282,7 +1282,7 @@
     {
         GdkPixbuf *pixbuf = gdk_pixbuf_new(GDK_COLORSPACE_RGB,
                                            gdk_pixbuf_get_has_alpha(GetPixbuf()),
-                                           8, GetWidth(), GetHeight());
+                                           8, rect.width, rect.height);
         ret.SetPixbuf(pixbuf);
         gdk_pixbuf_copy_area(GetPixbuf(),
                              rect.x, rect.y, rect.width, rect.height,
